hca/util/__init__.py

```diff
@@ -202,6 +203,25 @@ def __init__(self, config=None, **session_kwargs):
             for http_method, method_data in path_data.items():
                 self._build_client_method(http_method, http_path, method_data)
 
+    @staticmethod
+    def load_swagger_json(swagger_json, ptr_str="$ref"):
+        """
+        Load the Swagger JSON and resolve {"$ref": "#/..."} internal JSON Pointer references.
+        """
+        refs = []
+
+        def store_refs(d):
+            if len(d) == 1 and list(d)[0] == ptr_str:
```

Comment:
Using `next(iter(d))` or `ptr_str in d` for the second term is probably faster.

Reply:
Good idea, changed to `ptr_str in d`.